charts/lagoon-core/Chart.yaml

```diff
@@ -5,9 +5,9 @@ description: >
 home: https://github.com/uselagoon/lagoon-charts
 icon: https://raw.githubusercontent.com/uselagoon/lagoon-charts/main/icon.png
 maintainers:
-- name: smlx
-  email: scott.leggett@amazee.io
-  url: https://amazee.io
+  - name: smlx
```

Comment:
I think this might be a mistake.. could we change it back?

Reply:
otherwise looks good!

Reply:
Reverted changes, thanks!